### PR TITLE
Add support for multiline values

### LINKED ISSUE #4: Fails to read a configuration with long lines
The parser fails when reading the following file:

```
[bumpversion]
current_version = 0.0.1
commit = True
tag = True
parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(-a(?P<pmr>\d+)(\.dev(?P<build>\d+))?)?
serialize = 
	{major}.{minor}.{patch}-a{pmr}.dev{build}
	{major}.{minor}.{patch}-a{pmr}
	{major}.{minor}.{patch}

[bumpversion:file:setup.py]

[bumpversion:file:test/__init__.py]

[bumpversion:file:hpha_synchronization/__init__.py]

[bumpversion:file:hpha_synchronization/docs/__init__.py]

[bumpversion:file:hpha_synchronization/_internal/__init__.py]
```

on the lines following the `serialize` key. The same file reads fine using the python https://docs.python.org/3/library/configparser.html package.

### PR DESCRIPTION
**Pull request by dangmai**
This PR adds support for both reading and writing multiline values.

There are multiple examples in Python's configparser docs about this scenario: https://docs.python.org/3/library/configparser.html#supported-ini-file-structure
I've tried to handle as many edge cases as I can based off of those examples. I've also added tests for a few example cases.

Fixes https://github.com/ZachPerkitny/configparser/issues/4

### CODE REVIEW

File: src/configparser.js [hunk 1]
```diff
@@ -10,14 +10,19 @@ const interpolation = require('./interpolation');
  * @type {RegExp}
  * @private
  */
-const SECTION = new RegExp(/^\s*\[([^\]]+)]$/);
+const SECTION = new RegExp(/^(?<indent>\s*)\[(?<sectionName>[^\]]+)]$/);
```
Comment (dangmai) [inline src/configparser.js:13]: I switch the RegExp to use named groups, so that it's easier to reference them later in the code (vs referencing by index number)
